docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -408,6 +408,9 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+`TermsLookupQueryBuilder` has been removed, please use `TermsQueryBuilder` with no values and set the corresponding
```

Comment:
OK then it is a special case of our branch, I get it. Should we consider deprecating (or even removing it) though in master considering that it does the same thing as TermsQuery with lookup?